documentation/modules/deploying/proc-deploying-kafkaconnector.adoc

```diff
@@ -1,63 +1,96 @@
+The Cluster Operator updates the configurations of the running connector instances. 
+You remove a connector by deleting its corresponding `KafkaConnector`.
+
+`KafkaConnector` resources must be deployed to the same namespace as the Kafka Connect cluster they link to.
+
+In the configuration shown in this procedure, the `autoRestart` property is set to `true`.
+This enables automatic restarts of failed connectors and tasks. 
+Up to seven restart attempts are made, after which restarts must be made manually.
+You annotate the `KafkaConnector` resource to xref:proc-manual-restart-connector-str[restart a connector] or xref:proc-manual-restart-connector-task-str[restart a connector task] manually.
+
+NOTE: If you are using a Kafka version earlier than 3.0.0, only a failed connector is restarted and not failed tasks.
+
+.Example connectors
+
+You can use your own connectors or try the examples provided by Strimzi.
+Strimzi provides an xref:deploy-examples-{context}[example `KafkaConnector` configuration file] (`examples/connect/source-connector.yaml`).
+In this procedure, we show how to use the example file to create the following connector instances as `KafkaConnector` resources:
 
 * A `FileStreamSourceConnector` instance that reads each line from the Kafka license file (the source) and writes the data as messages to a single Kafka topic.
 * A `FileStreamSinkConnector` instance that reads messages from the Kafka topic and writes the messages to a temporary file (the sink).
 
 Alternatively, you can use the `examples/connect/kafka-connect-build.yaml` file to build a new Kafka Connect image with the file connectors. 
 
-Up until Apache Kafka 3.1.0, the example file connector plugins were included with Apache Kafka. Starting from the 3.1.1 and 3.2.0 releases of Apache Kafka, the examples need to be added to the plugin path as any other connector. See xref:using-kafka-connect-with-plug-ins-str[Extending Kafka Connect with connector plugins] for more details.
+Up until Apache Kafka 3.1.0, the example file connector plugins were included with Apache Kafka. Starting from the 3.1.1 and 3.2.0 releases of Apache Kafka, the examples need to be xref:using-kafka-connect-with-plug-ins-str[added to the plugin path as any other connector]. 
 
-[NOTE]
-====
-In a production environment, you prepare container images with the required Kafka Connect connectors, as described in xref:using-kafka-connect-with-plug-ins-{context}[].
-
-The `FileStreamSourceConnector` and `FileStreamSinkConnector` are provided as examples. Running these connectors in containers as described here is unlikely to be suitable for production use cases.
-====
+NOTE: In a production environment, you prepare container images with the required Kafka Connect connectors, as described in xref:using-kafka-connect-with-plug-ins-{context}[].
+Running the example connectors in containers as described here is unlikely to be suitable for production use cases.
```

Comment:
This changed the meaning of the previous warning quite a lot and it does not seem to make sense. It said that the example connectors are not production ready. Now you say that the wy to run them described here is not production ready. I think that is not the case. It should be either removed or reverted to the original meaning.

Reply:
I don't think we need what we have, but maybe a brief not to say something like this:

> NOTE: The example connectors are not intended for use in a production environment.